app/javascript/packs/component-definitions-common.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { addReact } from '../miq-component/helpers';
+import VmSnapshotFormComponent from '../components/vm-snapshot-form-component';
 
 /**
 * Add component definitions to this file.
 * example of component definition:
 * ManageIQ.component.addReact('ComponentName', ComponentName);
 */
+
+addReact('VmSnapshotForm', props => <VmSnapshotFormComponent {...props} />);
```

Comment:
should probably be just `addReact('VmSnapshotForm', VmSnapshotFormComponent)` now.

Reply:
..but it sounds confusing .. why the different name?